feat: add local registry mirror support for multiple CRIs (#206)
container-registry/harbor-satellite: QUICKSTART.md

            "rootDirectory": "./zot"
        },
        "http": {

>             "address": "127.0.0.1",

Reviewer: ```suggestion
            "address": "0.0.0.0",
```

revert this back.

```
> Note : by default, logging in JSON format is set to true. To change this pass additional flag `--json-logging=false`

### 7. Configure Local Registry as Mirror (Optional)

> Harbor Satellite allows you to set up a local registry as a mirror for upstream registries. Using the optional `--mirrors` flag, you can specify which upstream registries should be mirrored. The configured CRI will attempt to pull images from the local registry (Zot by default) first, and use the upstream registry as a fallback if the image is not available locally.

Reviewer: ```suggestion
Harbor Satellite allows you to set up a local registry as a mirror for upstream registries. Using the optional `--mirrors` flag, you can specify which upstream registries should be mirrored. The configured Container Runtime (CRI) will attempt to pull images from the local registry (Zot by default) first, and use the upstream registry as a fallback if the image is not available locally.
```

#### Supported CRIs
- `docker`
- `crio`
- `podman`
- `containerd`

#### Usage
```bash

> --mirrors=containerd:docker.io crio:docker.io

Reviewer: flags should accept only one args. Not many.

for example
satellite mirror1 mirror2 --mirrors=onething,twothing

> --mirrors=containerd:docker.io crio:docker.io

Reviewer: --mirrors=containerd:docker.io=https://mirror.gcr.io \
--mirrors=containerd:docker.io=https://my-local-mirror:5000

like the above we can try giving repeated flags for multiple CRIs

```

#### Notes

> - Docker: Only supports mirroring images from docker.io. Use `docker:true` to enable Docker mirroring. 

Reviewer: ```suggestion
- When using docker as a runtime it supports mirroring images from docker.io. So, use `--mirrors=docker:true` to enable Docker mirroring. 
```
